src/app/modules/submission/email-submission/email-submission.service.ts

```diff
@@ -254,3 +256,29 @@ export const saveSubmissionMetadata = (
     },
   )
 }
+
+export const sendSubmissionToAdmin = (
+  adminEmailParams: Parameters<typeof MailService['sendSubmissionToAdmin']>[0],
+): ResultAsync<true, SendAdminEmailError> => {
+  const errorLogParams = {
+    message: 'Error sending submission to admin',
+    meta: {
+      action: 'sendSubmissionToAdmin',
+      submissionId: adminEmailParams.submission.id,
+      formId: adminEmailParams.form._id,
+    },
+  }
+  return ResultAsync.fromPromise(
+    MailService.sendSubmissionToAdmin(adminEmailParams),
+    (error) => {
+      logger.error({ ...errorLogParams, error })
```

Comment:
eeehhhh about this, can't we just have a log meta and manually insert the message

Reply:
yeah I did this to accommodate the possibility of MailService returning false lol but now I've typed it to `true` instead of `boolean`